Dark_Heresy_2ed/DarkHeresy2ed.html

```diff
@@ -498,18 +498,222 @@
+				<div class="sheet-item" style="width: 15%;"><label>Damage:</label></div>
+				<div class="sheet-item" style="width: 20%;"><input name="attr_meleeweapondamage" type="text" /></div>
+				<div class="sheet-item" style="width: 12%;"><label>Type:</label></div>
+				<div class="sheet-item" style="width: 18%;"><input name="attr_meleeweapontype" type="text" /></div>
+				<div class="sheet-item" style="width: 10%;"><label>Pen:</label></div>
+				<div class="sheet-item" style="width: 10%;"><input name="attr_meleeweaponpen" type="text" value="0" /></div>
+				<div class="sheet-item" style="width: 16%;"><button name="roll_meleedamage" type="roll" value="/me does [[@{meleeweapondamage}+@{MutS}+(floor(@{strength}/10))]] @{meleeweapontype} damage! (Pen: @{meleeweaponpen})"> <label>Damage</label> </button></div>
+			</div>
+				<div class="sheet-row">
+				<div class="sheet-item" style="width: 15%;"><label>Special:</label></div>
+				<div class="sheet-item" style="width: 70%;"><input name="attr_meleeweaponspecial" type="text" /> </div>
+				<div class="sheet-item" style="width: 10%;"><button name="roll_meleehit" type="roll" value="/em uses @{meleeweaponname} [[1d100]] Target [[@{WeaponSkill}+ [[?{Aim | Half aim (+10),+10 | No aim (+0),+0 | Full aim (+20),+20} + ?{Attack Type | All out (+30),+30| Charge (+20),+20 |Standard (+10),+10 | Swift Attack (+0),+0 | Lighting (-10),-10} + ?{Modifier|0}]]]]."> <label>Hit</label> </button></div>
+			</div>
+		</fieldset>
+		<div class="sheet-quickborder">
+			<h3>Movement</h3>
+			<div class="sheet-row">
+				<div class="sheet-item" style="width: 20%;"><label>Half Move: </label></div>
+				<div class="sheet-item" style="width: 8%;"><input name="attr_HalfMove" type="number" value="floor(@{Agility}/10)+@{MutAg}+@{Size}" /></div>
+				<div class="sheet-item" style="width: 20%;"><label>Full Move: </label></div>
+				<div class="sheet-item" style="width: 8%;"><input name="attr_FullMove" type="number" value="2*(floor(@{Agility}/10)+@{MutAg}+@{Size})" /></div>
+				<div class="sheet-item" style="width: 16%;"><label>Charge: </label></div>
+				<div class="sheet-item" style="width: 8%;"><input name="attr_ChargeMove" type="number" value="3*(floor(@{Agility}/10)+@{MutAg}+@{Size})" /></div>
+				<div class="sheet-item" style="width: 10%;"><label>Run: </label></div>
+				<div class="sheet-item" style="width: 8%;"><input name="attr_RunMove" type="number" value="6*(floor(@{Agility}/10)+@{MutAg}+@{Size})" /></div>
+			</div>
+			<div class="sheet-row">
+				<div class="sheet-item" style="width: 28%;"><label>Fatigue</label></div>
+				<div class="sheet-item" style="width: 20%;"><label>Threshold:</label></div>
+				<div class="sheet-item" style="width: 8%;"><input name="attr_FatigueThreshold" type="number" /></div>
+				<div class="sheet-item" style="width: 16%;"><label>Current: </label></div>
+				<div class="sheet-item" style="width: 8%;"><input name="attr_Fatigue" type="number" /></div>
+			</div>
+	<br/>
+			<h3>Armour & Defence</h3>
```

Comment:
When you have an ampersand (&) in text in html you should use the html code: `&amp;`